### DISCUSSION TITLE
Restore Team or transfer domains

### DISCUSSION THREAD

**Discussion by ptenteromano** [vercel/community#6498] [Account Issue]
### Question

I went to downgrade from Pro back to hobby. To do so, I deleted my team. Apparently when I upgrade to pro all of my personal projects get migrated to the "team". Now they are lost. I'm looking to either restore the entirety of projects as they were or transfer the domains off of vercel.

**Comment by amyegan** [comment]
Hey @ptenteromano. If you let me know the domains, I can check on them for you.
